controllers/utils/managedclusterview.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	viewv1beta1 "github.com/stolostron/multicloud-operators-foundation/pkg/apis/view/v1beta1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const MCVLabelKey = "multicluster.odf.openshift.io/cluster"
```

Comment:
I don't think this is used anywhere...

Reply:
Yeah, we replaced with the createdBy label for MCO , i will remove it later